frontends/web/src/components/aopp/aopp.tsx

```diff
@@ -38,164 +37,114 @@ const Banner = ({ children }: TProps) => (
   <div className={styles.banner}>{children}</div>
 );
 
-type State = {
-  accountCode: accountAPI.AccountCode;
-  aopp?: aoppAPI.Aopp;
-}
-
-type Props = TranslateProps;
+const setAccountCodeDefault = (
```

Comment:
true, I generally like a more functional approach so the dependencies are clear, also technically this improves performance because the function is not defined/initialized on every render but performance does not really matter here xd.

I'm going to put it back in the component